include/field/launch_application.h

```diff
@@ -0,0 +1,439 @@
+#ifndef POKEHEARTGOLD_FIELD_LAUNCH_APPLICATION_H
+#define POKEHEARTGOLD_FIELD_LAUNCH_APPLICATION_H
+
+#include "bag.h"
+#include "battle/battle_setup.h"
+#include "script.h"
+#include "overlay_02.h"
+#include "overlay_63.h"
+#include "bag.h"
+#include "mail.h"
+#include "mail_misc.h"
+#include "fashion_case.h"
+#include "field_blackthorn_tutors.h"
+#include "pokemon_types_def.h"
+#include "save_palpad.h"
+#include "save_pokeathlon.h"
+#include "save_special_ribbons.h"
+#include "sav_system_info.h"
+#include "unk_0202CA24.h"
+#include "unk_02092BE8.h"
+#include "credits/credits.h"
+#include "voltorb_flip/voltorb_flip.h"
+
+typedef struct BerryPotsArgs {
```

Comment:
not sure how much I like the args structs all being in this file compared to their respective apps, it just makes *this* header very messy and makes it more difficult to reference the args when modifying individual app files.

Reply:
Agreed, moved the arg definitions out